bobtemplates/plone/view/views/+view_name_normalized+.pt.bob

```diff
@@ -0,0 +1,18 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="{{{ package.dottedname }}}"
+      metal:use-macro="context/main_template/macros/master">
+<body>
```

Comment:
check the file name here, it's using the wrong variable, we have the name for the template from a question, we should it ;)